documentation/docs/steps/gatlingExecuteTests.md

```diff
@@ -0,0 +1,23 @@
+# ${docGenStepName}
```

Comment:
I think you also have to register this file in another index file

Reply:
Thanks!